src/PackageCacheUploader/PackageCacheUploader.csproj

```diff
@@ -1,34 +0,0 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
It was used in Coherence-Final until we realized that we didn't want to be pushing nightly builds of our package cache to dist.asp.net. We stopped used it back in November